### PR TITLE
test: basic user flow test

### PR DESCRIPTION
**Pull request by rolsonquadras**
copied from https://github.com/trustbloc/sandbox/pull/1149/files

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

**Review by sudeshrshetty — APPROVED** [review]

File: test/ui-automation/test/helpers/wallet.js [hunk 1]
```diff
@@ -95,9 +128,10 @@ async function _getThirdPartyLogin(email) {
     return true;
   });
 
-  const signInButton = await $('#accept');
-  await signInButton.click();
+  const loginInButton = await $('#accept');
+  await loginInButton.click();
```
Comment (birtony) [inline test/ui-automation/test/helpers/wallet.js:132]: Maybe you meant `logIn`? In either way, aren't we using `signIn`/`signUp` syntax now?

Comment (rolsonquadras) [reply test/ui-automation/test/helpers/wallet.js:132]: yeah, makes sense. 

cc: @talwinder50

Comment (talwinder50) [reply test/ui-automation/test/helpers/wallet.js:132]: This loginin button is for login page which comes in popup we can rename to popupsignin may be

Comment (rolsonquadras) [reply test/ui-automation/test/helpers/wallet.js]: [renamed](https://github.com/trustbloc/wallet/compare/4a4426f351c82128fb4613e2c3aa0441bd1f1f28..74baac2b14e8c73755884b7e078dd88f7246bbb2) to `oidcLoginButton

**Review by birtony — APPROVED** [review]

**Pull request merged**